src/c.zig

```diff
@@ -113,10 +113,11 @@ pub fn moveFileZ(from_dir: std.os.fd_t, filename: [*:0]const u8, to_dir: std.os.
 }
 
 pub fn moveFileZWithHandle(from_handle: std.os.fd_t, from_dir: std.os.fd_t, filename: [*:0]const u8, to_dir: std.os.fd_t, destination: [*:0]const u8) !void {
+    _ = from_handle;
```

Comment:
In this scenario, `from_handle` is the file descriptor obtained from the `inject`, and it does not have read permissions.

https://github.com/oven-sh/bun/blob/3a9a6c63ac58564d8bae126a3edfc368ca4be671/src/standalone_bun.zig#L401

Should I directly modify the `inject` function to add read permissions?

Reply:
Yes

Reply:
Done.